docs/satellite-configuration-tuning.rst

```diff
@@ -229,6 +229,23 @@ Note:
 MongoDb Tuning
 ==============
 
+mongod consumes randomly high memory (up to 1/2 of all RAM) and this aggressive memory usage limits other processes or can cause OOM killers to kill mongod. In order to overcome this situation, tune the cache size by referring the following steps:
```

Comment:
randomly high - Are we sure that is random? If not I would rephrase to "under certain circumstances" or so

OOM killers => OOM killer

Reply:
Yes, i am not sure 100%, so changes according to the suggestions.